packages/lodestar-cli/src/lodecli/cmds/beacon/options/api.ts

```diff
@@ -0,0 +1,52 @@
+import {Options} from "yargs";
+
+export const apiEnabled: Options = {
+  alias: [
+    "api.enabled",
+    "api.rest.enabled",
```

Comment:
maybe also just "rest"

Reply:
It should be documented somewhere, but with these aliases, the first alias is the preferred name, (ideally consistently named across all options), the second alias is probably the option that `BeaconNode` expects as part of `IBeaconNodeOptions`.

Not opposed to adding more "convenience" aliases on top tho